addon/components/o-s-s/smart/input.stories.js

```diff
@@ -0,0 +1,68 @@
+import { hbs } from 'ember-cli-htmlbars';
+
+export default {
+  title: 'Components/OSS::Smart::Input',
+  component: 'button',
+  argTypes: {
```

Comment:
I think you can base off the argTypes from the `OSSInputContainer`.

A bunch of options are missing